app/javascript/components/patient/laboratory/FirstPositiveLab.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
```

Comment:
Would also love to see a test suite for this.  I know there aren't ones for the other labs files yet but since this one is new, it would be nice to have one added

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/ea0697f83c29320b8930fb7a5dcf63015ea342fb#diff-30c5340a7ce13eb30d2abf50df7d2d5821ef5e3ff48852f2a949a137fbc55774R1